sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -649,4 +691,17 @@ export class InstitutionController extends BaseController {
       totalUsers: institutionUserAndCount[1],
     };
   }
+  /**
+   * API to get the lookup values for admin role.
+   * @returns Admin Roles.
+   */
+  @IsInstitutionAdmin()
+  @Get("admin-roles")
+  async getAdminRoles(): Promise<UserRoleOptionDTO[]> {
+    const userRoles = await this.institutionService.getAdminRoles();
+    return userRoles.map((role) => ({
+      name: role.role || role.type,
```

Comment:
what if both `role` and `type` have value? only role need to consider? I am bit confused here

Reply:
also, can you add some comments about this logic

Reply:
Will add comments as discussed.

Reply:
comments added.